samplesheets/tests/test_views_taskflow.py

```diff
@@ -1090,9 +1091,14 @@ def setUp(self):
         self.assay_path = self.irods_backend.get_path(self.assay)
         self.path = os.path.join(self.assay_path, TEST_FILE_NAME)
```

Comment:
One further minor point: I would recommend renaming this and related paths to e.g. `self.obj_path*`. It specifies this is a path for a data object, not a collection. It makes the tests more readable.